src/pages/AdminPage/AdminCircleModal.tsx

```diff
@@ -450,6 +497,42 @@ export const AdminCircleModal = ({
           label="Auto Opt Out?"
         />
       </div>
+      <div style={{ display: 'grid' }}>
+        <p className={classes.subTitle}>Integrations</p>
+        <div className={classes.integrationContainer}>
+          {integrations.map((integration, index) => (
+            <div key={index} className={classes.integrationRow}>
+              <DeworkLogo size="md" className={classes.integrationIcon} />
+              <p className={classes.integrationText}>{integration.name}</p>
```

Comment:
This might be fine, give the well-communicated plans for V2, but it'd probably be good practice to encapsulate and abstract integration-specific components out of this parent component.